moped-editor/src/views/projects/projectView/ProjectView.js

```diff
@@ -254,9 +256,9 @@ const ProjectView = () => {
     setDialogContent(
       "Are you sure?",
       <span>
-        Cancelling this project will make it inaccessible to Moped users and
-        only available to administrators. Users may request a cancelled project
-        be restored by{" "}
+        Deleting this project will make it inaccessible to Moped users and only
+        available to administrators. Users may request a delete project be
```

Comment:
should this be "deleted project" ? I can update in my branch, just wanted to confirm it was a typo and not intended phrasing

Reply:
Sweet, that would be amazing. Thanks @chiaberry

Reply:
There's actually an issue for this text that @tillyw has: https://github.com/cityofaustin/atd-data-tech/issues/7837